cfn-resources/search-deployment/cmd/resource/mappings.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource
+
+import "go.mongodb.org/atlas-sdk/v20231115007/admin"
+
+func newCFNSearchDeployment(prevModel *Model, apiResp *admin.ApiSearchDeploymentResponse) Model {
```

Comment:
is CFN useful in the name or we could remove it?

Reply:
Based the naming from terraform repo convention. I do feel it is helpful for readability, if removed the method would be `newSearchDeployment` (SDK to CFN) which would be very similar to `newSearchDeploymentReq` (CFN to SDK). Open to opinions here. cc: @oarbusi @maastha as similar conversation might rise.